views/templates/hook/order-confirmation.tpl

```diff
@@ -13,16 +13,16 @@
 *}
 {if $paypal_mx_order.valid == 1}
 <div class="conf confirmation">
-	{l s='Congratulations! Your payment is pending verification, and your order has been saved under' mod='paypalmx'}{if isset($paypal_mx_order.reference)} {l s='the reference' mod='paypalmx'} <b>{$paypal_mx_order.reference|escape:html:'UTF-8'}</b>{else} {l s='the ID' mod='paypalmx'} <b>{$paypal_mx_order.id|escape:html:'UTF-8'}</b>{/if}.
+	{l s='Felicidades! Su pago está en proceso de verificación, y su orden ha sido guardada con el número' mod='paypalmx'}{if isset($paypal_mx_order.reference)} {l s='de referencia' mod='paypalmx'} <b>{$paypal_mx_order.reference|escape:html:'UTF-8'}</b>{else} {l s='the ID' mod='paypalmx'} <b>{$paypal_mx_order.id|escape:html:'UTF-8'}</b>{/if}.

```

Comment:
Please write the content in English first, and then translate it in your favorite language.